config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
Not sure what you are asking. The way the plugin works in this PR is:
1. User configures the plugin to work with some directory path
2. User creates a volume that defines a remote host + key:
`docker volume create -d vieux/sshfs -o sshcmd=<user@host:path> -o identity=<identity_file_name> <volumename>`
3. user bind-mounts that volume to a container

If the user wants to work with additional remote hosts (using the same or possibly a different key), they can repeat steps 2 and 3